v1 (#36)
cosmos/new-cosmos.network: content/articles/what-is-staking.md

---
index: 19
title: What is staking?

> description: Staking is the process of locking up a digital asset (ATOM in the case of the Cosmos Hub) in order to provide economic security for a public blockchain.

Reviewer: ```suggestion
description: Staking is the process of locking up a digital asset (ATOM in the case of the Cosmos Hub) to provide economic security for a public blockchain.
```

  - blockchain
  - atom
---

> Staking is the process of locking up a digital asset (ATOM in the case of the Cosmso Hub) in order to provide economic security for a public blockchain. Public blockchains are permissionless networks, meaning anyone is free to participate in maintaining them. As a result, it is possible for some of the maintainers of the network (called [validators](/learn/faq/what-is-a-validator) to act maliciously. In order to incentivise maintainers to behave in the best interest of the network, the locked up assets are at risk of being partially slashed (i.e. destroyed) if evidence is brought that a fault was committed.

Reviewer: ```suggestion
Staking is the process of locking up a digital asset (ATOM in the case of the Cosmso Hub) to provide economic security for a public blockchain. Public blockchains are permissionless networks, meaning anyone is free to participate in maintaining them. As a result, it would be possible for some of the maintainers of the network (called [validators](/learn/faq/what-is-a-validator) to act maliciously. To incentivise maintainers to behave in the best interest of the network, the locked-up assets are at risk of being partially slashed (destroyed) if there is evidence that a fault was committed.
```
